iquip/apps/builder.py

```diff
@@ -320,16 +324,33 @@ def __init__(self, name: str, argInfo: Dict[str, Any], parent: Optional[QWidget]
               ndecimals: The number of displayed decimals.
         """
         super().__init__(name, argInfo, parent=parent)
-        self.state = self.get_state()
-        procdesc = self.get_procdesc()
-        # TODO(AIJUH): Add features for stack widget.
-        self.stackWidget = QStackedWidget(self)
-        self.rangeWidget = _RangeScan(procdesc, self.state["RangeScan"])
+        self.state = self.stateManager()
+        procdesc = self.procdescManager()
+        self.stackedWidget = QStackedWidget(self)
+        buttonLayout = QHBoxLayout()
+        self.scanButtonGroup = QButtonGroup(self)
+        scanDict = {"NoScan": (_NoScan, "No scan"), "RangeScan": (_RangeScan, "Range")}
+        self.scanWidgetsDict = {}
```

Comment:
It looks sufficient to name it to `scanWidgets`.
In other apps, we have hardly indicated "Dict".